src/generators/minecraft-mutant-character/MutantCharacter.res

```diff
@@ -0,0 +1,507 @@
+
+let id = "mutantgenerator"
+
+let name = "Mutant Character"
+let video: Generator.videoDef = {url: "https://www.youtube.com/embed/DVzumgRinjY"}
+let thumbnail: Generator.thumnbnailDef = {
+  url: Generator.requireImage("./thumbnail/thumbnail.jpeg"),
+}
+let images: array<Generator.imageDef> = [
+  {id: "Page1", url: requireImage("Page1.png")},
+  {id: "Page2", url: requireImage("Page2.png")},
+  {id: "Page3", url: requireImage("Page3.png")},
+  {id: "Page4", url: requireImage("Page4.png")},
+  {id: "Hole1", url: requireImage("Hole1.png")},
+  {id: "hole2", url: requireImage("hole2.png")},
+  {id: "hole3", url: requireImage("hole3.png")},
+  
+]
+
+let textures: array<Generator.textureDef> = [
+  {
+    id: "Skin",
+    url: requireTexture("Skin.png"),
+    standardWidth: 64,
+    standardHeight: 64,
+  }
+]
+
+let script = () => {
+  
+Generator.defineTextureInput("Skin", {
+        standardWidth: 64,
+        standardHeight: 64,
+        choices: []
+      });
+
+
+Generator.defineSelectInput("Color", [
+        "Steve",
+        "Alex",
+        
+        
+      ]);
+let alexModel = Generator.getSelectInputValue("Color") === "Alex";
+
+
+Generator.usePage("Page 1");
+Generator.drawImage("Page1", (0, 0));
+// Head
+Generator.drawTextureLegacy(
+  "Skin",
+  { x: 0, y: 8, w: 8, h: 8 },
+  { x: 260, y: 88, w: 84, h: 84 },()
+); // Right
+Generator.drawTextureLegacy(
+  "Skin",
+  { x: 8, y: 8, w: 8, h: 8 },
+  { x: 2, y: 88, w: 84, h: 84 },()
+); // Face
+Generator.drawTextureLegacy(
+  "Skin",
+  { x: 16, y: 8, w: 8, h: 8 },
+  { x: 88, y: 88, w: 84, h: 84},()
+); // Left
+Generator.drawTextureLegacy(
+  "Skin",
+  { x: 24, y: 8, w: 8, h: 8 },
+  { x: 174, y: 88, w: 84, h: 84 },()
+  
+  
+); // Back
+Generator.drawTextureLegacy(
+  "Skin",
+  { x: 8, y: 0, w: 8, h: 8 },
+  { x: 2, y: 2, w: 84, h: 84 },()
+); // Top
+Generator.drawTextureLegacy(
+  "Skin",
+  { x: 16, y: 0, w: 8, h: 8 },
+  { x: 2, y: 174, w: 84, h: 84 },
+  ~flip=#Vertical,
+  (),
+); // Bottom
+
+
+Generator.drawTextureLegacy(
+    "Skin",
+    { x: 32, y: 8, w: 8, h: 8 },
+    { x: 260, y: 88, w: 84, h: 84 },()
+  ); // Right
+  Generator.drawTextureLegacy(
+    "Skin",
+    { x: 40, y: 8, w: 8, h: 8 },
+    {  x: 2, y: 88, w: 84, h: 84},()
+  ); // Face
+  Generator.drawTextureLegacy(
+    "Skin",
+    { x: 48, y: 8, w: 8, h: 8 },
+    { x: 88, y: 88, w: 84, h: 84 },()
+  ); // Left
+  Generator.drawTextureLegacy(
+    "Skin",
+    { x: 56, y: 8, w: 8, h: 8 },
+    {  x: 174, y: 88, w: 84, h: 84 },()
+  ); // Back
+  Generator.drawTextureLegacy(
+    "Skin",
+    { x: 40, y: 0, w: 8, h: 8 },
+    { x: 2, y: 2, w: 84, h: 84  },()
+  ); // Top
+  Generator.drawTextureLegacy(
+    "Skin",
+    { x: 48, y: 0, w: 8, h: 8 },
+    { x: 2, y: 174, w: 84, h: 84 },
+    ~flip=#Vertical,
+  (),
+  ); // Bottom
+  
+  ////Right Arm
+  
+  if (alexModel) {
+      Generator.drawTextureLegacy( "Skin", { x: 40, y: 20, w: 4, h: 6  }, {x:200, y:330, w:64, h:168}, 
```

Comment:
Formatting needs to be fixed up. As mentioned above, install the `rescript-vscode` extension inside VSCode and it should format the files correctly for you.

Reply:
Done